src/micromamba/install.sh

```diff
@@ -1,41 +1,50 @@
 #!/usr/bin/env bash
 
 VERSION=${VERSION:-"latest"}
+# Add destination parameter here?
 
-USERNAME=${USERNAME:-"automatic"}
+# # This looks like it's not needed:
+# USERNAME=${USERNAME:-"automatic"}
 
 set -e
 
+# Is this the standard for dev-container features? It seems like this way each
+# feature will rerun `apt-get update` which seems inefficient. But I suppose
+# this is to prioritize reliability?
 # Clean up
 rm -rf /var/lib/apt/lists/*
 
 if [ "$(id -u)" -ne 0 ]; then
     echo -e 'Script must be run as root. Use sudo, su, or add "USER root" to your Dockerfile before running this script.'
+    # Or it could install at the user level?
     exit 1
 fi
 
+# Do we really need this check? Other architectures are theoretically suppored, namely ppc64le.
+# Also, should we be using `uname` instead of `dpkg`?
```

Comment:
Of course it can support other architectures.

Reply:
Would you object to removing this check? This way, we always try and download a release for the current platform, so we wouldn't be responsible for synchronizing this check with the list of supported architectures.

Reply:
I would be happy to remove this check. I simply cannot guarantee it works as I am unable to test it.